rpcserver.go

```diff
@@ -8431,3 +8435,23 @@ func rpcInitiator(isInitiator bool) lnrpc.Initiator {
 
 	return lnrpc.Initiator_INITIATOR_REMOTE
 }
+
+func (r *rpcServer) Quiesce(_ context.Context, in *lnrpc.QuiescenceRequest) (
+	*lnrpc.QuiescenceResponse, error) {
+
+	txid, err := lnrpc.GetChanPointFundingTxid(in.ChanId)
+	if err != nil {
+		return nil, err
+	}
+
+	op := wire.NewOutPoint(txid, in.ChanId.OutputIndex)
+	cid := lnwire.NewChanIDFromOutPoint(*op)
```

Comment:
This looks like an unnecessary round trip: we go from chan ID, to chan point, then back to chan ID...

Reply:
It's actually not, due to the incompatible types from the rpc (`in.ChanID` is, indeed, *not* a `ChannelID`)